pyproject.toml

```diff
@@ -33,11 +33,18 @@ dependencies = [
   "ipywidgets",
   "pyee",
   "pyserial",
-  "pyserial-asyncio"
+  "pyserial-asyncio",
+]
+
+[project.optional-dependencies]
+dbus = [
+  "sdbus",
 ]
 
 [project.scripts]
 plusdeck = "plusdeck.cli:main"
+plusdeckd = "plusdeck.dbus.service:main"
```

Comment:
It would be good to create a `plusdeck` package that depends on `python-plusdeck` which includes a systemd unit file. If rpm's can configure users, that may be valuable to do as well.